CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 0.5.6
+
+* Update tax and benefit system initialization method in scripts.
```

Comment:
And that did not raise the compatible minimum version number of `openfisca-france`?